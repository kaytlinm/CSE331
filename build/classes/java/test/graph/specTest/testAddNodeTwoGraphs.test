#Add Same Node to Two Graphs Test File
#Tests that two graphs can be created and that the same node can be added to the graphs separately

CreateGraph graph1
CreateGraph graph2
AddNode graph1 node1
AddNode graph2 node1