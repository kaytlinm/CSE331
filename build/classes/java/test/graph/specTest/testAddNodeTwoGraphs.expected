#Add Same Node to Two Graphs Test File
#Tests that two graphs can be created and that the same node can be added to the graphs separately

created graph graph1
created graph graph2
added node node1 to graph1
added node node1 to graph2
